### DISCUSSION TITLE
[u-boot challenge] docker build failed sometimes

### DISCUSSION THREAD

**Discussion by xooxo** [github/securitylab#76] [General] [answered]
Hi all,

First of all, I'm excited about learning capabilities of codeQL. Great idea,great implementation!
I took U-Boot course but when commit fails for some reason (that being docker build failure most of the time), github-learning-bot stops working. I mean even if my other commits give true answers, course does not move on. I deleted repo and started again but same happens on Step 3 this time.(It had stopped before on Step 4)
What should I do? I'm getting tired of deleting my repo and start all over again. 

I tried closing issue manually, too, by the way

**Comment by xcorail** [comment, marked as answer]
Hey 👋🏾 
Can you please give me read access to your repo?

**Reply by xcorail** [reply]
@xooxo do you still need help?

**Reply by xooxo** [reply]
yeah i do,sorry,i forgot.

**Reply by xooxo** [reply]
I invite you as a collabarator. Is that okay?

**Reply by xcorail** [reply]
Seems like you don't have the problem so far 🤔 
Let me know when you have reproduced the issue.

**Reply by xooxo** [reply]
yes,interestig enough, i pass step 4 this time. I deleted and re-created repo :)

**Reply by xcorail** [reply]
Ok, let me know if it happens again 🤔

**Reply by xooxo** [reply]
Sure! Am i able to close now and re-open this discussion if problem occurs again?

**Reply by xcorail** [reply]
Sure.
